MonoGame.Framework/Platform/Native/Audio.Interop.cs

```diff
@@ -0,0 +1,131 @@
+// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using Microsoft.Xna.Framework.Audio;
+using System;
+using System.Runtime.InteropServices;
+
+
+namespace MonoGame.Interop;
+
+
+
+[MGHandle]
+internal readonly struct MGA_System { }
+
+[MGHandle]
+internal readonly struct MGA_Buffer { }
+
+[MGHandle]
+internal readonly struct MGA_Voice { }
+
+
+/// <summary>
+/// MonoGame native calls for platform audio features.
+/// </summary>
+internal static unsafe partial class MGA
```

Comment:
@mrhelmut @harry-cpp  

Because i needed it on Switch, i went ahead and created the native audio API.  This API is based on how the console APIs worked with a audio system, buffers, and voices.

Later we will have to figure out how FAudio will be integrated under this API but also move to using FAudio's Xact implementation.